udfs/remote_udfs/nlp/README.md

```diff
@@ -0,0 +1,119 @@
+# Calling Google's Natural Language API Sentiment Analysis Example 
+
+Google's Natural Language API Sentiment Analysis inspects the given text and identifies the prevailing emotional opinion within the text, especially to determine a writer's attitude as positive, negative, or neutral.  
+
+To run this example, you will need the following APIs enabled:
+* [BigQuery API](https://cloud.google.com/bigquery/docs)
+* [Cloud Functions API](https://cloud.google.com/functions/docs)
+* [BigQuery Connections API](https://cloud.google.com/bigquery/docs/working-with-connections#enable_the_connection_service)
+* [Natural Language API](https://cloud.google.com/natural-language/docs/setup#api)
+
+## Running the example
+
+### Setting the environment variables 
+
+Replace the various environment variables below with your desired values.
+```
+PROJECT=your_project_id
+LOCATION=your_bigquery_dataset_location
+DATASET=your_bigquery_dataaset_id
```

Comment:
typo: your_bigquery_dataset_id

It seems that we assume it exists and it's in the location, instead should we include the dataset creation command in the location.

Reply:
Updated 

We do assume it exists, we already re-use the location, I think it's safe enough - since there are a few nuances (we need to make sure the dataset doesn't exist, check location etc.)

Reply:
Probably call it out, followed a create dataset bq cmd just in case. In this example, the dataset is only used for hosting the remote function and there's limitation on location, so creating a new dataset seems more convenient for audience.